tests/mocks/step.py

```diff
@@ -5,7 +5,10 @@
 
 
 class MockStep:
```

Comment:
```suggestion
class DummyStep:
```

Also consider using the naming "Stub" or "Fake", both may be representative of what we try to accomplish here. Relevant article: https://jesusvalerareales.com/testing-with-test-doubles/

Also, does not need to be solved here, you can make an issue for it and suggest a renaming for all similar "Mocks" that should be called "Dummy", "Stub", or "Fake"

Reply:
Good idea, and great tip for a relevant resource. I will defer this to a separate issue, and have [created one](https://github.com/equinor/isar/issues/524).